C_CPP/Operating System Concepts/non-preemptive/non-preemptive_scheduling/non-preemptive-scheduling.md

```diff
@@ -0,0 +1,103 @@
+# **_Operating System Concepts: Non-Preemptive Scheduling_**
+
+# **OVERVIEW**
+
+In non pre-emptive scheduling, once the Cpu is assigned to a process, the process keeps the CPU until it either terminates or switches to the waiting state.
+
+This method of scheduling was used by Windows 3.x
+
+### Let us discuss some non pre-emptive scheduling algorithms in detail:
+
+# **FIRST COME FIRST SERVE (FCFS)**
+
+### Scheduling Criteria: Arrival time
+
+### Mode: Non preemptive
+
+Here, the Cpu is allocated on first come first serve basis, i.e, the process that requests Cpu first is allocated the CPU first. It can be managed using a First-in First-out(FIFO) queue.
+
+## Code: [click here for code!](fcfs.cpp)
+
+## OUTPUT:
+
+![output](fcfs_output.png)
```

Comment:
![image](https://user-images.githubusercontent.com/78724676/135743287-77129621-f10b-4502-87b6-2da169dc8302.png)
fix this as well

Reply:
none of your output are rendered in "non-preemptive-scheduling.md" this file. Fix this before your next PR.